nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -92,32 +93,29 @@ class MRDeGibbsInputSpec(MRTrix3BaseInputSpec):
         mandatory=True,
         desc='input DWI image')
     axes = traits.ListInt(
-        default_value=[0,1],
-        usedefault=True,
+        [0,1],
```

Comment:
There's no point to a default value if `usedefault` isn't set. We should either keep this or get rid of it entirely.


```suggestion
        default_value=[0, 1],
        usedefault=True,
```

```suggestion
```

Reply:
I believe these would work without keyword arg, although it is better being explicit.